Bottom: 3bb7c3078c8b400d724f9027c535b957f4f41556
Top:    99c46884957a3cf4faab8ad7f13ef86b0cd673e3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 00:02:39 +0100

Require GTK 2.18 or later for the GTK HID.

This is for some new widgets, like the GtkInfoBar.


---

diff --git a/configure.ac b/configure.ac
index 422347a..20b5cd3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -752,8 +752,8 @@ for e in $HIDLIST; do
 		AC_MSG_ERROR([Cannot find pkg-config, make sure it is installed and in your PATH])
 	fi
 
-	PKG_CHECK_MODULES(GTK, gtk+-2.0 >= 2.12.0, ,
-		[AC_MSG_ERROR([Cannot find gtk+ >= 2.12.0, install it and rerun ./configure
+	PKG_CHECK_MODULES(GTK, gtk+-2.0 >= 2.18.0, ,
+		[AC_MSG_ERROR([Cannot find gtk+ >= 2.18.0, install it and rerun ./configure
 Please review the following errors:
 $GTK_PKG_ERRORS])]
 	)
